File: langgraph/result/qwen/7b-py2js/0.js
function hasCloseElements(numbers, threshold) {
    for (let i = 0; i < numbers.length; i++) {
        for (let j = 0; j < numbers.length; j++) {
            if (i !== j) {
                let distance = Math.abs(numbers[i] - numbers[j]);
                if (distance < threshold) {
                    return true;
                }
            }
        }
    }

    return false;
}

File: langgraph/outputs/equiv/0.py
from typing import List
import itertools

def contains_nearby_pairs(values: List[float], max_distance: float) -> bool:
    for first, second in itertools.combinations(values, 2):
        if abs(first - second) < max_distance:
            return True
    return False
# method name: has_close_elements -> contains_nearby_pairs
# parameter names: numbers -> values, threshold -> max_distance
# replaced nested loops with itertools.combinations for pair generation
# used descriptive variable names first/second instead of idx/idx2
# simplified distance check with direct comparison